source/main.lua

```diff
@@ -8,7 +8,7 @@
 ]]
 
 -- variables for use with testing/debugging:
--- DEBUG_GAME = "jira" --> Set "DEBUG_GAME" variable to the name of a minigame and it'll be chosen every time!
+-- DEBUG_GAME = "minigame_kunstofweg" --> Set "DEBUG_GAME" variable to the name of a minigame and it'll be chosen every time!
```

Comment:
Deze moet weggie als we em willen mergen

Reply:
Oh ik dacht dat het kunst was :(